Backend/README.md

```
@@ -1,34 +1,35 @@
+- User authentication via Firebase (Signup/Login)
+- Store additional user data in MongoDB
+- Uses Express.js for the backend
+- Mongoose for MongoDB integration
+- Frontend built with React.js
 
-Store additional user data in MongoDB
+## Installation & Setup
 
-Uses Express.js for the backend
+### 1. Clone the Repository
+```sh
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)